### PR TITLE
Various things to get passing tests

### PR DESCRIPTION
**Pull request by eefahy**
This is the first step to getting an autodeploy working for the website environment migration

### CODE REVIEW

**Comment by azaroth42** [comment]
Looks okay to me, but prefer @jpstroop to check the ruby. :+1: for what it's worth.

**Comment by jpstroop** [comment]
Sorry, have been away. I'll have a look ASAP.

File: Gemfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 source 'https://rubygems.org'
 
 gem 'rake'
-gem 'jekyll', '2.0.3'
-gem 'kramdown', '1.3.3'
+gem 'jekyll', '3.4.3'
+gem 'kramdown', '1.12.0'
```
Comment (jpstroop) [inline Gemfile]: https://github.com/IIIF/iiif.io/issues/1149 ✔️

**Comment by jpstroop** [comment]
:+1: Happy to merge unless @glenrobson wants to pull the trigger.

**Pull request merged**